packages/decode-lz-options/src/components/outputOptions.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Box, Text } from "ink";
+import { printJson } from "@layerzerolabs/io-devtools";
+import {
+  ComposeOption,
+  ExecutorLzReceiveOption,
+  ExecutorNativeDropOption,
+} from "@layerzerolabs/lz-v2-utilities";
+
+type OutputOptionsType = {
+  composeOptions: ComposeOption;
+  nativeDropOptions: ExecutorNativeDropOption;
+  lzReceiveOption: ExecutorLzReceiveOption | undefined;
+  orderedOption: boolean;
+};
+
+export const OutputOptions: React.FC<{ config: OutputOptionsType }> = ({
+  config,
+}) => {
+  return (
+    <Box flexDirection="column">
+      <Text>
+        <Text bold>Result:</Text>
+        {"\n"}
+        <Text color="green">{printJson(config)}</Text>
+      </Text>
```

Comment:
Tiny thing but you can drop the wrapper `Text` and the `{"\n"}` and you should get the same result (maybe add a `gap={1}` on the `Box` to get an empty line.

Reply:
`gap` is not available;  I used `padding={1}` instead.